frontend/src/components/OverviewForm.tsx

```diff
@@ -0,0 +1,27 @@
+import {
+  Box,
+  Text,
+  Avatar,
+  AvatarGroup
+} from "@chakra-ui/react";
+
+import { Assignee } from "utils/types";
+
+// Overview Form component for displaying forms in the dashboard
+// will probably have to change the types once the backend is finished
+export const OverviewForm = ({ formName, assignees } : { formName: String, assignees: Array<Assignee> }) => {
+  return <>
+      <Box w={308} h={136} border="1px solid #C0C0C0" borderRadius={5}>
+        <Box paddingLeft="28px" paddingTop="28px">
+          <Text fontFamily="Helvetica" fontWeight={700} fontSize="22px">
+            {formName}
+          </Text>
+          <AvatarGroup size="md" max={5} marginTop={19}>
+            {assignees.map((assignee: Assignee, index: number) => {
+              return <Avatar name={assignee.name} key={index} boxSize="36px" backgroundColor={assignee.signed ? "#BBDFC8" : "#FFFFFF"} border="1px solid #A8A8A8" color="black"/>
```

Comment:
Looks great, just a minor nitpick. Per the figma, can the border around the signed profiles be a darker green instead of the gray? I think the background of the signed ones should be "#D0F0DC" and the border is "#BBDFC8"